### ISSUE TITLE
[🐞] V2: Important types like `SVGAttributes` are not exported

### ISSUE THREAD

**Issue by fabian-hiller** [QwikDev/qwik#7212]
### Which component is affected?

Qwik Runtime

### Describe the bug

In some components like [this one](https://github.com/fabian-hiller/valibot/blob/main/website/src/icons/AngleRightIcon.tsx#L3) I need to access the generated JSX types but they are not exported in the build of Qwik core in `dist/core-internal.d.ts`.

### Reproduction

https://stackblitz.com/edit/github-o2f6fj9n?file=src%2Fcomponents%2FCheckIcon.tsx

### System Info

```shell
-
```

**Comment by JerryWu1234** [comment]
What expectation do you want ?

**Comment by fabian-hiller** [comment]
I expect to be able to import Qwik's JSX TS types somehow. Otherwise, I am forced to manually copy them into my project.

**Comment by wmertens** [comment]
@fabian-hiller care to make a PR? It's just a few lines and running API.update

**Comment by JerryWu1234** [comment]
> @fabian-hiller care to make a PR? It's just a few lines and running API.update

I could do it.

**Comment by Varixo** [comment]
the `PropsOf<'svg'>' isn't the same?

**Comment by fabian-hiller** [comment]
@JerryWu1234 feel free to do so. I have a lot going on at the moment.

**Comment by fabian-hiller** [comment]
> the PropsOf<'svg'> isn't the same?

This works. Is this planned as an official solution? When I hover over a specific attribute it shows me `SVGAttributes<Element>`. That's why I thought I need to import this type somehow.

![Screenshot ](https://github.com/user-attachments/assets/7c54e325-6ede-4bf1-b25f-fb6ffd939997)

**Comment by JerryWu1234** [comment]
@fabian-hiller @Varixo Should I close this issue?

**Comment by fabian-hiller** [comment]
Depends on how the Qwik team responds to my previous comment.

**Comment by wmertens** [comment]
Yes I suppose we need to export this type even though PropsOf is nice

**Comment by JerryWu1234** [comment]
this was finished
